source/settings/plugin/acl.rst

```diff
@@ -254,3 +263,18 @@ List cache
 ``dovecot-acl-list`` file lists all mailboxes that have ``l`` rights assigned.
 If you manually add/edit ``dovecot-acl`` files, you may need to delete the
 ``dovecot-acl-list`` to get the mailboxes visible.
+
+Dictionaries
+^^^^^^^^^^^^
+
+In order for an ACL to be fully useful, it has to be communicated to IMAP clients.
+For example, if you use ACL to share a mailbox to another user, the client has to
+be explicitly told to check out the other user's mailbox too, as that one is shared.
+Placing the ACL file makes the ACL effective, but ``dovecot`` doesn't take care of
+the user->shared mailboxes mapping out of the box, and as a result, it won't publish
+shared mailboxes to clients if this is not set up.
+You have to configure this manually by defining an appropriate
+`dictionary <https://wiki.dovecot.org/Dictionary>` to store the share map.
```

Comment:
needs `_` at the end.

Reply:
Fixed, along with the other place where I copied this from.